Shamaz.xcodeproj/project.pbxproj

```diff
@@ -12,7 +12,10 @@
 		D02ACE18242808BA003938D1 /* ViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = D02ACE17242808BA003938D1 /* ViewController.swift */; };
 		D02ACE1B242808BA003938D1 /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = D02ACE19242808BA003938D1 /* Main.storyboard */; };
 		D02ACE1D242808BC003938D1 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = D02ACE1C242808BC003938D1 /* Assets.xcassets */; };
-		D02ACE20242808BC003938D1 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = D02ACE1E242808BC003938D1 /* LaunchScreen.storyboard */; };
+		D02D1AAB2431579900DE9B6F /* phrases.swift in Sources */ = {isa = PBXBuildFile; fileRef = D02D1AAA2431579900DE9B6F /* phrases.swift */; };
```

Comment:
Any reason for the file name to start with a lower case?
This comment also apply for  `design.swift`

Reply:
No particular reason. I had a problem renaming it with a capital letter after creating it so I had to rename it to phrases2 and then -> Phrases. Weird XCode behavior!